src/index.js

```diff
@@ -24,8 +12,28 @@ const {
   PROFILE_MEMORY,
 } = process.env
 
+const chalk = require("chalk")
+console.log(chalk.green(`\n[Force] NODE_ENV=${NODE_ENV}\n`))
+
+if (NODE_ENV === "development") {
+  require("coffeescript/register")
+  require("@babel/register")({
+    extensions: [".ts", ".js", ".tsx", ".jsx"],
+    plugins: ["babel-plugin-dynamic-import-node"],
+  })
+}
+
+global.Promise = require("bluebird")
```

Comment:
Is this necessary? I mean, I know it's already used, buuut... do we need it?

Reply:
Feel free to open a PR ;)

Reply:
Fair point